Tests/js/views/assets/ez-translator.js

```diff
@@ -0,0 +1,11 @@
+YUI.add('ez-translator', function (Y) {
```

Comment:
I would move this file in `Tests/js/assets/` as it will be used all over the place for various components (not only views).